person.rb

```diff
@@ -1,11 +1,19 @@
 require './nameable'
 class Person < Nameable
+  attr_reader :rentals
```

Comment:
- Getters and Setters for @rentals should be present. this means that these 2 methods `attr_reader` and `attr_writer` for rental should be present. Therefore, instead of writing both methods, you can use the shortcut `attr_accessor`
- That is why you need to Apply what the previous review said.  
- Open this 👉 [ Link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor) to learn more through examples.
- NB: USING THIS METHOD 👉 `attr_accessor`  IS OPTIONAL. if you don't need it, then both  `attr_reader` and `attr_writer` should be present